src/scripts/core/visualization-container.js

```diff
@@ -74,6 +74,12 @@
             return this.ctx;
         },
 
+        setVisibility: function (visible) {
+            var node = this.layer.node();
+
+            visible ? node.style.display = 'block' : node.style.display = 'none';
```

Comment:
I'm not sure display will _always_ be block for svg elements (See circle in screenshot of contour dom for e.g.), but if you're sure it will be for this specific case that's good enough

<img width="1734" alt="screen shot 2017-01-20 at 2 49 17 pm" src="https://cloud.githubusercontent.com/assets/830274/22168158/b34790ba-df1f-11e6-8c48-734743333caa.png">